python/lsst/meas/mosaic/fluxfit.cc

```diff
@@ -0,0 +1,79 @@
+/*
+ * LSST Data Management System
+ * Copyright 2008-2017  AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https:
+ */
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include "lsst/meas/mosaic/fluxfit.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace mosaic {
+
+PYBIND11_PLUGIN(fluxfit) {
+    py::module::import("lsst.afw.cameraGeom");
+    py::module::import("lsst.afw.image");
+    py::module::import("lsst.daf.base");
+
+    py::module mod("fluxfit");
+
+    py::class_<FluxFitParams, std::shared_ptr<FluxFitParams>> cls(mod, "FluxFitParams");
+
+    cls.def_readwrite("order", &FluxFitParams::order);
+    cls.def_readwrite("chebyshev", &FluxFitParams::chebyshev);
+    cls.def_readwrite("ncoeff", &FluxFitParams::ncoeff);
+    cls.def_readwrite("xorder", &FluxFitParams::xorder, py::return_value_policy::copy);
+    cls.def_readwrite("yorder", &FluxFitParams::yorder, py::return_value_policy::copy);
+    cls.def_readwrite("absolute", &FluxFitParams::absolute);
+    cls.def_readwrite("coeff", &FluxFitParams::coeff, py::return_value_policy::copy);
```

Comment:
`xorder`, `yorder`, and `coeff` seem to be used as arrays, so I'm not sure this wrapping will work for those.  I'm fairly confident those could never be used from Python and it's probably best just to not wrap them at all.